kubernetes-deployment-yaml

```diff
@@ -0,0 +1,47 @@
+apiVersion: apps/v1 # for versions before 1.9.0 use apps/v1beta2
+kind: StatefulSet
```

Comment:
Any reason this is a StatefulSet instead of a Deployment? If not I find Deployments easier to work with.